pkg/config/nerdctl_config_applier.go

```diff
@@ -124,6 +126,10 @@ func updateNerdctlConfig(fs afero.Fs, user string, rootless bool) error {
 	}
 
 	cfg.Namespace = nerdctlNamespace
+	cfg.HostGatewayIP = "192.168.5.2"
```

Comment:
instead of having this magic string down here, could we define as a constant `defaultNerdctlHostGatewayIP` here https://github.com/runfinch/finch/pull/453/files#diff-531afb9f97a7384e0de895046f9e14befbebc49c896b080fce4c41c75f14f1b4L20-L21

then you can also reference it in unit tests where you expect the `HostGatewayIP` to be the default value.